IntroToCSharp/Pages/Activities/Variables.razor

```diff
@@ -0,0 +1,148 @@
+        computer has the capacity to “remember” a lot of things. Computers can remember numbers, letters, whole words,
+        and a lot more. However, they can't remember things that the programmer doesn’t explicitly tell them to. So,
+        when we
+        are coding we must use variables to label and store bits and pieces of essential data that we will need later.
+    </MudText>
+</Section>
+<Section Title="The Basics of Variables">
+    <MudText>
+        A variable is a temporary piece of stored data within the computer's memory.
+    </MudText>
+    <Example Title="Home Address">
+        Your personal belongings are stored in your home but in order to find your house you need the home address. The
+        same is true for a variable. It has an address that allows us to access and alter its data.
+    </Example>
+</Section>
+
+<Section Title="What Makes Up A Variable?">
+    <MudList Clickable="true">
+        <MudListItem Icon="@Icons.Material.Filled.Badge" IconColor="Color.Secondary" Text="Name">
+            <NestedList>
+                <MudListItem>
+                    <MudText>This is the “address” that you declare that allows you to access the variable later.
+                    </MudText>
+                </MudListItem>
+            </NestedList>
+        </MudListItem>
+        <MudListItem Icon="@Icons.Material.Filled.Category" IconColor="Color.Warning" Text="Type">
+            <NestedList>
+                <MudListItem>
+                    <MudText>The type refers to the data being stored. Data can be a bunch of different things so it’s
+                        important for the computer to know what type of data it is handling.
+                    </MudText>
+                    <Example Title="Ice Cream Truck">
+                        You are not going to go up to the ice cream truck and just say you want ice cream
+                        because the vender won’t know what flavor to give you. Instead, you tell him what flavor or type
+                        of ice cream you want. That way he can go straight to the freezer and get strawberry ice cream.
+                    </Example>
+                </MudListItem>
+            </NestedList>
+        </MudListItem>
+        <MudListItem Icon="@Icons.Material.Filled.Storage" IconColor="Color.Tertiary" Text="Data">
+            <NestedList>
+                <MudListItem>
+                    <MudText>Data is the actual information stored.</MudText>
+                    <Example Title="Fun Fact">
+                        Let's say that you memorize the date when the United States declared its
+                        independence. The "data" stored in your brain would be "July 4, 1776".
+                    </Example>
+                </MudListItem>
+            </NestedList>
+        </MudListItem>
+    </MudList>
+
+</Section>
+
+<Section Title="Using Variables">
```

Comment:
I would like a section called "Recipe for Reading Data"

To read data, you simply state the variable name. This tells the computer to "substitute" the value stored by that variable into the code. This process is called "evaluation". For example:

```csharp
int z = x + y;
```
When the computer tried to run the code above, it must first "evaluate"  `x` and `y` before it can determine what `z` is. Let's assume that `x` stores the number `5` and `y` stores the number `7`. During evaluation, the computer will substitute those values in like this:
```csharp
int z = 5 + 7;
```

It will then add them together:

```csharp
int z = 12;
```

And finally, it will store the value 12 in the variable `z`.